apps/files/src/components/File-Details-Buttonrow.vue

```diff
@@ -40,10 +40,31 @@
 </template>
 <script>
     export default {
+        props : ['files'],
         data () {
             return {
                 fileAction: null
             }
+        },
+        methods: {
+            download() {
+                console.log("cat")
+                console.log(this.files)
+                for(let i = 0; i < this.files.length; i++){
+                    OC.$client.files.getFileContents(this.files[i].path).then(res => {
```

Comment:
This is Just wrong. Let the browser handle the download.

Reply:
Oh sorry I've missed some debug code.. 
Can you clarify on "Let the browser handle the download" please? I thought this is how we allow the download to happen through browser.

Reply:
Basically an anchor tag with the url to the File should be enough.

Reply:
That is what I have done. I have to call the *getFileContents* multiple times when the user has selected multiple files to be downloaded. As its current state, js-owncloud-client API doesn't support downloading multiple files with a single call. And then I let the browser handle each response using and anchor tag. 

If I am still not approaching that correctly, please advise.